rdr_service/api/genomic_api.py

```diff
@@ -29,3 +29,20 @@ def get(self, mode=None, p_id=None):
             return self._make_response(proto_payload)
 
         raise BadRequest
+
+
+class GenomicOutreachApi(BaseApi):
+    def __init__(self):
+        super(GenomicOutreachApi, self).__init__(GenomicOutreachDao())
+
+    @auth_required(RDR_AND_PTC)
+    def get(self, mode=None):
+        if mode not in ('GEM', 'RHP'):
+            raise BadRequest("GenomicOutreach Mode required to be \"GEM\" or \"RHP\".")
+
+        if mode == "GEM":
+            report_states_result = self.dao.date_lookup(mode)
+
+            return self._make_response(report_states_result)
+
+        return BadRequest
```

Comment:
Ultra compact: "return self._make_response(self.dao.date_lookup(mode)) if mode == "GEM" else BadRequest"

Reply:
Yes, I'll compact this, thanks.

Reply:
I just realized I had moved this into another method.